### PR TITLE
wayfire.ini: update config

### PR DESCRIPTION
**Pull request by travankor**

### CODE REVIEW

File: wayfire.ini [hunk 1]
```diff
@@ -132,14 +135,11 @@ outputs = kanshi
...
-# http://jonls.dk/redshift/
-# Note: Requires Redshift with Wayland support.
-# https://github.com/jonls/redshift/pull/663
-redshift = redshift -m wayland
+# https://sr.ht/~kennylevinsen/wlsunset/
+gamma = wlsunset
 
 # Idle configuration
 # https://github.com/swaywm/swayidle
-# https://github.com/swaywm/swaylock
```
Comment (wb9688) [inline wayfire.ini:142]: Why is this lined removed?

Comment (travankor) [reply wayfire.ini:142]: you can set up swayidle from a config file now, plus there's a separate section for swaylock

Comment (ammen99) [reply wayfire.ini:142]: well the example itself uses swaylock so I think the link should remain there.

**Review by ammen99 — APPROVED** [review]
Thanks!

**Pull request merged**